feat: op naar productie

marnixdessing opened the pull request:
Latest version of the Yivi issue app, ready for production

marnixdessing committed:
fix: flip condition to make demo default

Commit:
chore: self mutation

Signed-off-by: github-actions <github-actions@github.com>

marnixdessing committed:
Merge pull request #48 from GemeenteNijmegen/feat/yivi

fix: flip condition to make demo default

marnixdessing committed:
fix: keep using irma-auhtorization header

marnixdessing committed:
fix: logging and docs

marnixdessing committed:
fix: use existing irma schema

marnixdessing committed:
fix: failure in qr code

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/3963569131

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #24 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/3972310344

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #49 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/3977254684

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #50 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/3982264017

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #51 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/3991792513

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #52 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4001648067

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #53 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4011129514

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #54 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4020359067

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #55 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4029225066

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #56 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4034259901

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #57 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4039377759

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #58 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4049462733

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #61 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4059627792

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #62 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4070000386

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #63 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

joostvanderborg committed:
feat: New frontend

Started a new frontend. Removed all old image resources and css-files.
Created a new main.css, added yivi-logo (2 versions). Als added a
temporary static html-file for iterating on the design. login function
uses new resources. The other pages will probably be broken currently.

Commit:
chore: self mutation

Signed-off-by: github-actions <github-actions@github.com>

marnixdessing committed:
Merge branch 'acceptance' into feat/yivi-frontend

marnixdessing committed:
fix: remove snapshot from git index

Commit:
chore: self mutation

Signed-off-by: github-actions <github-actions@github.com>

marnixdessing committed:
Merge pull request #64 from GemeenteNijmegen/feat/yivi-frontend

feat: yivi frontend

marnixdessing committed:
feat: html into new frontend

marnixdessing committed:
Merge branch 'feat/yivi-frontend' of github.com:GemeenteNijmegen/yivi-issue-app into feat/yivi-frontend

marnixdessing committed:
Merge pull request #65 from GemeenteNijmegen/feat/yivi-frontend

feat: yivi frontend html

joostvanderborg committed:
feat: Styling improvements

joostvanderborg committed:
feat: Change title

marnixdessing committed:
Merge pull request #66 from GemeenteNijmegen/feat/yivi-frontend

feat: yivi frontend styling and buttons

marnixdessing committed:
feat: nice buttons

marnixdessing committed:
Merge pull request #67 from GemeenteNijmegen/feat/yivi-frontend

feat: nice buttons

marnixdessing committed:
feat: nice buttons

marnixdessing committed:
Merge pull request #68 from GemeenteNijmegen/feat/yivi-frontend

feat: nice buttons

marnixdessing committed:
feat: nicer error messages in frontend

marnixdessing committed:
Merge pull request #69 from GemeenteNijmegen/feat/yivi-frontend

feat: nicer error messages in frontend

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4079885775

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #70 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

marnixdessing committed:
feat: YIVI -> Yivi

marnixdessing committed:
fix: use lare logo by default

marnixdessing committed:
fix: improve language

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4088974985

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #71 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4094106539

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #72 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4099183745

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #73 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4109208808

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #74 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4119498915

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #75 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4129662546

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #76 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4139634645

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #77 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

joostvanderborg committed:
feat: Remove unneccessary pseudo-element and span

Move the digid background into the button background. Remove necessity
of span element.

joostvanderborg committed:
feat: Added primary, digid, and regular styles

The logout link is no longer styled as a digid/primary button. This is
not the secondary action, and the logout button does not require the
logo.

joostvanderborg committed:
Merge pull request #78 from GemeenteNijmegen/feat/main-screen

feat: Button changes

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4148740435

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #79 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4153831469

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #80 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4158972612

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #83 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

joostvanderborg committed:
feat: Transition background

To communicatie hover state, the background transitions (in size)
on hover.

marnixdessing committed:
feat: security.txt

marnixdessing committed:
Merge branch 'acceptance' of github.com:GemeenteNijmegen/yivi-issue-app into acceptance

marnixdessing committed:
feat: webpage meta data

marnixdessing committed:
fix: additional meta data text

marnixdessing committed:
fix: front-end styling

marnixdessing committed:
fix: open graph metadata

marnixdessing committed:
feat: set bot control to count to allow meta data indexing and search engines

marnixdessing committed:
feat: setup yivi.accp.nijmegen.nl

marnixdessing committed:
fix: nederland -> Nederland

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4169142258

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #87 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4179339435

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #88 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4189404529

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #89 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4199441023

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #90 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4208654529

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #91 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4213699340

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #92 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4218899863

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #93 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

marnixdessing committed:
feat: certificate validation with alternative names

marnixdessing committed:
Merge branch 'acceptance' of github.com:GemeenteNijmegen/yivi-issue-app into acceptance

marnixdessing committed:
feat: undeploy cloudfront stack

marnixdessing committed:
feat: redeploy cloudfront stack

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4228075740

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #95 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

marnixdessing committed:
feat: use powertools for additional logging

marnixdessing committed:
Merge pull request #98 from GemeenteNijmegen/feat/auth-lambda-logging

feat: use powertools for additional logging

marnixdessing committed:
feat: min loa implemented

marnixdessing committed:
Merge pull request #99 from GemeenteNijmegen/feat/auth-lambda-logging

feat: min loa implemented

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4238095215

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #100 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4248208529

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #101 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4258088510

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #102 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4267356973

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #103 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4272513669

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #104 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4277690160

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #105 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4288181131

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #106 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4298705195

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #107 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4309298026

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #108 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4319494593

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #109 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4328356148

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #111 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4333545742

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #112 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4338754577

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #113 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4349311392

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #114 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4359849088

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #115 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4369796973

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #116 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4380053435

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #117 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4389562567

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #118 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4394864418

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #119 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4400186617

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #120 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

marnixdessing committed:
Merge branch 'acceptance' into feat/transition-bg

marnixdessing committed:
fix: added background size back to css

marnixdessing committed:
Merge pull request #86 from GemeenteNijmegen/feat/transition-bg

feat: transition bg

marnixdessing committed:
fix: primary button not working

marnixdessing committed:
Merge pull request #126 from GemeenteNijmegen/feat/transition-bg

fix: primary button not working

marnixdessing committed:
fix: primary button not working

marnixdessing committed:
feat: dynamically load ssm parameters (and eslint)

marnixdessing committed:
feat: use getParameter in YiviApi

marnixdessing committed:
feat: use getParameter to obtain ssm parameters in issue lambda

marnixdessing committed:
Merge pull request #128 from GemeenteNijmegen/feat/dynamically-load-parameters

feat: dynamically load parameters (issue lambda)

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4410824223

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #129 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

marnixdessing committed:
feat: OIDC utility uses getParameter and fixed tests

marnixdessing committed:
fix: rename env variables to *_SSM

marnixdessing committed:
feat: added refresh button

marnixdessing committed:
feat: enforce digid level of assurance in oidc scope

marnixdessing committed:
Merge pull request #130 from GemeenteNijmegen/feat/dynamically-load-parameters

feat: OIDC utility uses getParameter and fixed tests

marnixdessing committed:
fix: granded read rights to wrong lambda

marnixdessing committed:
Merge pull request #131 from GemeenteNijmegen/feat/dynamically-load-parameters

fix: granded read rights to wrong lambda

marnixdessing committed:
fix: button spacing

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4421448770

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #132 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

marnixdessing committed:
fix: missing hidden css class

marnixdessing committed:
fix: update session on callback only when success

marnixdessing committed:
feat: enable paring support

invalid-email-address committed:
chore: self mutation

Signed-off-by: github-actions <github-actions@github.com>

marnixdessing committed:
Merge pull request #135 from GemeenteNijmegen/feat/dynamically-load-parameters

feat: frontend improvements

marnixdessing committed:
feat: add yivi server response to frontend

marnixdessing committed:
feat: use yivi session response in frontend

marnixdessing committed:
Merge branch 'acceptance' into feat/yivi-session-pairing

marnixdessing committed:
Merge pull request #136 from GemeenteNijmegen/feat/yivi-session-pairing

feat: yivi session pairing

marnixdessing committed:
fix: wrong base64 decode and options endpoint

marnixdessing committed:
Merge pull request #137 from GemeenteNijmegen/feat/yivi-session-pairing

fix: wrong base64 decode and options endpoint

marnixdessing committed:
fix: clean up old session pointer variables

marnixdessing committed:
fix: wrong endpoint

marnixdessing committed:
Merge pull request #138 from GemeenteNijmegen/feat/yivi-session-pairing

fix: clean up and set correct endpoint

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4432226951

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #139 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

marnixdessing committed:
feat: include DigiD LOA in Yivi attribute

marnixdessing committed:
fix: added additinoal LOA tests

marnixdessing committed:
fix: added additinoal LOA tests

marnixdessing committed:
feat: increase max statistics collection to 1.5 years

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4442868613

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #140 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4452677490

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #141 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4458023362

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #142 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4463470082

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #143 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

marnixdessing committed:
fix: digidlevel should be string not number

marnixdessing committed:
Merge branch 'acceptance' of github.com:GemeenteNijmegen/yivi-issue-app into acceptance

marnixdessing committed:
fix: debug log session data

marnixdessing committed:
fix: do not try to show qr if no session is provided

marnixdessing committed:
fix: failing test

marnixdessing committed:
fix: remove logging of session data

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4474210667

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #144 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

marnixdessing committed:
feat: handle deceased response from BRP api

marnixdessing committed:
Merge branch 'acceptance' of github.com:GemeenteNijmegen/yivi-issue-app into acceptance

marnixdessing committed:
fix: log error instead of log

marnixdessing committed:
fix: depricated function, statment order

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4485068686

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #146 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4495727495

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #147 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

marnixdessing committed:
feat: added additional tests

marnixdessing committed:
feat: more testing

marnixdessing committed:
Merge branch 'acceptance' of github.com:GemeenteNijmegen/yivi-issue-app into acceptance

marnixdessing committed:
chore: self mutate

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4506464694

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #148 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

marnixdessing committed:
docs: test bevindingen

marnixdessing committed:
Merge branch 'acceptance' of github.com:GemeenteNijmegen/yivi-issue-app into acceptance

marnixdessing committed:
fix: improved and checked logging statements

marnixdessing committed:
feat: removed lambda powetools logger from auth lambda

marnixdessing committed:
feat: added production configuration

marnixdessing committed:
fix: make YiviApi class region independent

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4516426748

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #149 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4521891020

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #150 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4527390763

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #151 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4537888834

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #153 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

joostvanderborg commented on src/Configuration.ts:
```diff
@@ -42,6 +43,17 @@ export interface Configuration {
+
+  /**
+   * If the issue lambda uses the demo scheme or the production scheme.
+   * Note: shoud only be true in production
```
should **not** be true in production?

joostvanderborg commented on src/ParameterStage.ts:
```diff
@@ -57,7 +57,7 @@ export class ssmParamsConstruct extends Construct {
 
     new SSM.StringParameter(this, 'ssm_auth_3', {
-      stringValue: 'openid idp_scoping:simulator idp_scoping:https://was-preprod1.digid.nl/saml/idp/metadata',
+      stringValue: 'openid idp_scoping:simulator service:DigiD_Midden idp_scoping:https://was-preprod1.digid.nl/saml/idp/metadata',
```
Als het goed is werkt `idp_scoping:digid` ipv `idp_scoping:https://was-preprod1.digid.nl/saml/idp/metadata` nu ook. Daarmee is preprod en prod gelijk qua settings (behalve simulator dan in dit geval).

joostvanderborg commented on src/UsEastCertificateStack.ts:
```diff
@@ -18,16 +25,36 @@ export class UsEastCertificateStack extends Stack {
       parameterName: Statics.certificateArn,
     });
 
+    if (configuration.cnameRecords) {
```
Wat gebeurt hier? Ik zou er in ieder geval graag een verklarende comment bij zien (of extracten naar eigen functie).

joostvanderborg commented on src/WafStack.ts:
```diff
@@ -50,52 +51,52 @@ export class WafStack extends Stack {
           },
         },
         // After counting the SignalNonBrowserUserAgent matches, block all except the excluded ua
-        {
```
Als we dit verwijderen is r. 45 ook niet meer nodig denk ik? 
```
excludedRules: [
                {
                  name: 'SignalNonBrowserUserAgent',
                },
              ],```

joostvanderborg reviewed (commented):
Gedeeltelijke review, not niet alles doorgenomen.

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4548914854

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #154 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4559242039

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #155 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

joostvanderborg reviewed (approved):
Gezamenlijk codereview gedaan, nog wat aandachtspunten maar niks wat merge tegenhoudt.
 👍

marnixdessing committed:
chore: mark points of improvement as todo

marnixdessing committed:
docs: added documentation image of all lambdas

marnixdessing committed:
Merge branch 'acceptance' of github.com:GemeenteNijmegen/yivi-issue-app into acceptance
